benchmark/sirun/dogstatsd.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const dgram = require('dgram')
+
+class DogStatsD {
+  constructor (options) {
+    options = options || {}
+
+    this._prefix = options.prefix || ''
+    this._tags = options.tags || []
+    this._buffer = ''
+    this._udp = this._socket('udp4')
+  }
+
+  gauge (stat, value, tags) {
+    this._add(stat, value, 'g', tags)
+  }
+
+  increment (stat, value, tags) {
+    this._add(stat, value, 'c', tags)
+  }
+
+  flush () {
+    const buffer = this._buffer
+
+    if (buffer.length === 0) return
+
+    this._buffer = ''
+
+    this._udp.send(buffer, 0, buffer.length, 8125)
+  }
+
+  _add (stat, value, type, tags) {
+    const message = `${this._prefix + stat}:${value}|${type}`
+
+    tags = tags ? this._tags.concat(tags) : this._tags
```

Comment:
tags are not currently supported by sirun

Reply:
Removed tags as they are DogStatsD specific anyway.